### PR TITLE
doc:add config for hot region history and add command for pd ctl

### PR DESCRIPTION
**Pull request by qidi1**
Signed-off-by: qidi1 <1083369179@qq.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add config for hot region history and add command for pd ctl

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v5.4 (TiDB 5.3 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- English Version: pingcap/docs/pull/6627
- Other reference link(s):
    Relative docs:
    * Chinese vergion: pingcap/docs-cn/pull/7164
    * English vergion: pingcap/docs/pull/6624
    
    Feature tracking Issue: 
    * pingcap/tidb/pull/25281

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@qidi1 Please perform the following tasks for this PR. Thanks!

- Sign the CLA as reminded by https://github.com/pingcap/docs-cn/pull/7163#issuecomment-926306539.
- Specify the applicable version in the PR description part. If it is a new config, it is recommended to specify in which version the config is introduced.

File: pd-control.md [hunk 1]
```diff
@@ -491,7 +491,7 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 ]
 ```
 
-### `hot [read | write | store]`
+### `hot [read | write | store| history]`
```
Comment (JmPotato) [inline pd-control.md:494]: I can't find any PR that adds this new command into the `pd-ctl`, it seems that it has not been developed yet?

image

File: pd-configuration-file.md [hunk 2]
```diff
@@ -277,6 +277,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 是否使用 Joint Consensus 进行副本调度。关闭该特性时，PD 将采用一次调度一个副本的方式进行调度。
 + 默认值：true
 
+### `hot-regions-write-interval`
+
+* 设置pd存储hot Region信息时间间隔。
```
Comment (JmPotato) [inline pd-configuration-file.md:282]:
```suggestion
* 设置 PD 存储 Hot Region 信息时间间隔。
```

Please take care of the typography.

File: pd-configuration-file.md [hunk 3]
```diff
@@ -277,6 +277,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+* 默认值：10m
+
+> 注意：
+>
+> hot region的信息一般三分钟更新一次，如果设置时间间隔小于三分钟，中间部分更新可能没有意义。
+
+### `hot-regions-reserved-days`
+
+* 设置pd保留多久之前的hot Region信息
```
Comment (JmPotato) [inline pd-configuration-file.md:291]: Ditto.

**Review by JmPotato — COMMENTED** [review]
Please indicate the versions affected by these changes also, I think it's master though.

File: pd-control.md (hunk 1 above)
Comment (qidi1) [reply pd-control.md:494]: it`s not merge yet

File: pd-configuration-file.md (hunk 3 above)
Comment (qidi1) [reply pd-configuration-file.md]: changed

File: pd-configuration-file.md (hunk 2 above)
Comment (qidi1) [reply pd-configuration-file.md]: changed

File: pd-configuration-file.md [hunk 4]
```diff
@@ -277,6 +277,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 是否使用 Joint Consensus 进行副本调度。关闭该特性时，PD 将采用一次调度一个副本的方式进行调度。
 + 默认值：true
 
+### `hot-regions-write-interval`
+
+* 设置 pd 存储 Hot Region 信息时间间隔。
```
Comment (JmPotato) [inline pd-configuration-file.md:282]:
```suggestion
* 设置 PD 存储 Hot Region 信息时间间隔。
```

File: pd-configuration-file.md [hunk 5]
```diff
@@ -277,6 +277,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+* 默认值：10m
+
+> 注意：
+>
+> hot region的信息一般三分钟更新一次，如果设置时间间隔小于三分钟，中间部分更新可能没有意义。
+
+### `hot-regions-reserved-days`
+
+* 设置 pd 保留多久之前的 Hot Region 信息
```
Comment (JmPotato) [inline pd-configuration-file.md:291]:
```suggestion
* 设置 PD 保留 Hot Region 信息的最长时间
```

File: pd-control.md [hunk 6]
```diff
@@ -519,6 +519,50 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 
+显示历史读写热点信息:
+
+{{< copyable "" >}}
+
+```
+>> hot history startTime endTime [ <name> <value> ]
+```
+
+例如查询时间`1634478065`到`1734478065`之间的历史hot_region信息:
```
Comment (JmPotato) [inline pd-control.md:530]:
```suggestion
例如查询时间 `1634478065` 到 `1734478065` 之间的历史 hot_region 信息:
```

File: pd-control.md [hunk 7]
```diff
@@ -519,6 +519,50 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+    QueryRate     ... 
+    StartKey      ... 
+    EndKey        ... 
+    EncryptionMeta ...
+  }
+  ......
+]
+```
+
+对于参数的值为数组的请用[x,y,...]形式进行参数值的设置，如下所示:
```
Comment (JmPotato) [inline pd-control.md:560]:
```suggestion
对于参数的值为数组的请用 [x, y, ...] 形式进行参数值的设置，如下所示:
```

Comment (qidi1) [reply pd-control.md]: apply change to code

File: pd-control.md (hunk 6 above)
Comment (qidi1) [reply pd-control.md]: apply change to code

File: pd-configuration-file.md (hunk 5 above)
Comment (qidi1) [reply pd-configuration-file.md]: apply change to code

File: pd-configuration-file.md (hunk 4 above)
Comment (qidi1) [reply pd-configuration-file.md]: apply change to code

**Review by nolouch — APPROVED** [review]
lgtm

File: pd-configuration-file.md [hunk 8]
```diff
@@ -275,6 +275,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+* 默认值：10m
+
+> 注意：
+>
+> Hot Region 的信息一般 3 分钟更新一次。如果设置时间间隔小于 3 分钟，中间部分的更新可能没有意义。
+
+### `hot-regions-reserved-days`
+
+* 设置 PD 保留的 Hot Region 信息的最长时间。
+* 默认值: 7days
```
Comment (hfxsd) [inline pd-configuration-file.md:290]: 我根据另外一篇文档的描述，把默认值改成 7days 。请确认下 @qidi1

Comment (hfxsd) [inline pd-configuration-file.md:290]:
```suggestion
* 默认值: 7
```

File: pd-configuration-file.md [hunk 9]
```diff
@@ -275,6 +275,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+* 设置 PD 存储 Hot Region 信息时间间隔。
+* 默认值：10m
+
+> 注意：
+>
+> Hot Region 的信息一般 3 分钟更新一次。如果设置时间间隔小于 3 分钟，中间部分的更新可能没有意义。
+
+### `hot-regions-reserved-days`
+
+* 设置 PD 保留的 Hot Region 信息的最长时间。
```
Comment (hfxsd) [inline pd-configuration-file.md:289]:
```suggestion
* 设置 PD 保留的 Hot Region 信息的最长时间。单位为天。
```

**Comment by qidi1** [comment]
增加了一个commit, 把`store_id`, `region_id`, `peer_id` 的 `[1,2,3]` 输入方法纠正成 `1,2,3`。

File: pd-control.md [hunk 10]
```diff
@@ -511,6 +511,77 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+>> hot history startTime endTime [ <name> <value> ]
+```
+
+例如查询时间 `1629294000000` 到 `1631980800000` （毫秒）之间的历史热点 Region 信息:
+
+{{< copyable "" >}}
+
+```
+>> hot history 1629294000000 1631980800000
+
```
Comment (TomShawn) [inline pd-control.md:528]:
```suggestion
```

Comment (TomShawn) [inline pd-control.md:528]:
```suggestion
```

File: pd-configuration-file.md [hunk 11]
```diff
@@ -275,6 +275,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 是否使用 Joint Consensus 进行副本调度。关闭该特性时，PD 将采用一次调度一个副本的方式进行调度。
 + 默认值：true
 
+### `hot-regions-write-interval`
```
Comment (TomShawn) [inline pd-configuration-file.md:278]:
```suggestion
### `hot-regions-write-interval` <span class="version-mark">从 v5.4.0 版本开始引入</span>
```

File: pd-configuration-file.md [hunk 12]
```diff
@@ -275,6 +275,20 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+
+* 设置 PD 存储 Hot Region 信息时间间隔。
+* 默认值：10m
+
+> **注意：**
+>
+> Hot Region 的信息一般 3 分钟更新一次。如果设置时间间隔小于 3 分钟，中间部分的更新可能没有意义。
+
+### `hot-regions-reserved-days`
```
Comment (TomShawn) [inline pd-configuration-file.md:287]:
```suggestion
### `hot-regions-reserved-days` <span class="version-mark">从 v5.4.0 版本开始引入</span>
```

File: pd-control.md [hunk 13]
```diff
@@ -511,6 +511,76 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+      "query_rate": 305,
+      "start_key": "7480000000000000FF5300000000000000F8",
+      "end_key": "7480000000000000FF5600000000000000F8"
+    },
+    ...
+  ]
+}
+```
+
+对于参数的值为数组的请用 x, y, ... 形式进行参数值的设置，所有支持的参数如下所示:
```
Comment (TomShawn) [inline pd-control.md:553]:
```suggestion
对于参数的值为数组的请用 `x, y, ...` 的形式进行参数值的设置，所有支持的参数如下所示:
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**